news/2021-04-08-monthly-beatmapping-contest-april-2021.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "Monthly Beatmapping Contests: April 2021"
+date: 2021-04-08 19:00:00 +0000
+---
+
+Compete with other mappers to find out who holds the crown in this slidershape-themed mapping contest!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+This month's [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) (*MBC*) includes opportunities for **osu!**, **osu!taiko**, and **osu!catch** mappers. If you're an osu!mania mapper, next month is your time to shine.
+
+## Schedule
+
+- **Beatmapping (2.5 weeks):** Contests are announced at the start of the month and beatmappers create their entries.
+- [**Screening (1.5 weeks):**](/wiki/Contests/Monthly_Beatmapping_Contest#screening) Competitive entries are selected by experienced beatmappers for each mode.
+- [**Judging (1.5 weeks):**](/wiki/Contests/Monthly_Beatmapping_Contest#judging) Competitive entries are judged by different experienced beatmappers for each mode.
+- [**Rewards:**](/wiki/Contests/Monthly_Beatmapping_Contest#rewards) Winners are announced and prizes are redeemed, including fancy profile badges and the legendary *Elite Mapper* title!
+
+Because this process takes about 6 weeks, results will be announced halfway through next month. That said, March's contest is about to enter the judging phase. Look forward to its outcome soon!
+
+## Beatmapping
+
+### <a id="limitation"></a>Limitation
+
+To ensure each MBC is a brand new experience, they all feature a **limitation** — a mapping restriction that all submissions must follow, like *no slider velocity changes* or *circles cannot be used*.
+
+**[osu! MBC](https://osu.ppy.sh/community/contests/122) limitations:**
+
+- All sliders must use red anchors.
+- Red anchors must form clear corners in the slidershape.
+
+This idea leaves room for interpretation, so here's a few examples of acceptable sliders. Note how each of their red anchors create a distinct bump in the slidershape:
+
+![](/wiki/shared/news/2021-04-08-monthly-beatmapping-contest-april-2021/allowed.png)
+
+And here's a few examples of what is *NOT* allowed. They either have no red anchors, or their red anchors don't create corners in the slidershape:
+
+![](/wiki/shared/news/2021-04-08-monthly-beatmapping-contest-april-2021/not_allowed.png)
+
+Because of the obvious potential for loopholes with this limitation, we'll give more weight to "limitation" category in this month's judging (similar to the [overlap-themed MBC](https://osu.ppy.sh/home/news/2020-09-08-monthly-beatmapping-contest-september) from a few months ago).
```

Comment:
```suggestion
Because of the obvious potential for loopholes with this limitation, we'll give more weight to the "limitation" category in this month's judging (similar to the [overlap-themed MBC](https://osu.ppy.sh/home/news/2020-09-08-monthly-beatmapping-contest-september) from a few months ago).
```